src/tests/Interop/MarshalAPI/FunctionPointer/GenericFunctionPointer.cs

```diff
@@ -8,7 +8,6 @@
 
 public partial class FunctionPtr
 {
-    public static bool CanRunGenericFunctionPointerTest => !TestLibrary.Utilities.IsMonoRuntime;
     public static bool CanRunInvalidGenericFunctionPointerTest => !TestLibrary.Utilities.IsNativeAot && !TestLibrary.Utilities.IsMonoRuntime;
```

Comment:
Both mono and NativeAOT don't have an error mode for runtime marshalled generic calli, so we keep skipping the negative tests for them.